recipes/qiskit-ignis/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "qiskit-ignis" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/qiskit-ignis-{{ version }}.tar.gz
+  sha256: c2351fe6c479b64c39d2786da3d0f15db122611cee039874a8979c7e583b72a5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy >=1.13
+    - python >=3.6
+    - qiskit-terra >=0.15.1
+    - retworkx >=0.8.0
+    - scipy >=0.19,!=0.19.1
+    - setuptools >=40.1.0
+
+test:
+  imports:
+    - docs
```

Comment:
Docs should not be installed.

Reply:
Ahh shoot thanks. I'll upstream that.

Reply:
CI is correctly failing now: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=502148&view=logs&jobId=6f142865-96c3-535c-b7ea-873d86b887bd&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823